community/bolt/pom.xml

```diff
@@ -163,11 +163,11 @@
       <artifactId>websocket-client</artifactId>
       <scope>test</scope>
     </dependency>
-      <dependency>
-          <groupId>commons-lang</groupId>
-          <artifactId>commons-lang</artifactId>
-          <version>RELEASE</version>
-          <scope>test</scope>
-      </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>RELEASE</version>
```

Comment:
i think while we here we should change it to particular version number or even better would be to switch it to commons-lang3 which other modules use

Reply:
Will create a card for this to cleanup common-lang dependecies